### DISCUSSION TITLE
Cannot change Billing Information

### DISCUSSION THREAD

**Discussion by Lippiece** [community/community#84139] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't see any way to edit my billing information:
![image](https://github.com/community/community/assets/47634624/6d140a13-53f0-41c9-82d9-819afd7dfccb)

**Comment by darkotodoric** [comment, marked as answer]
If your original address is from Russia, that could be the reason. Unfortunately, GitHub doesn't allow you to edit the mentioned information because of this. Try contacting them by email if you no longer live in Russia.

**Reply by Lippiece** [reply]
I see. Quite frustrating.
